include/ibm/management_console_rest.hpp

```diff
@@ -485,13 +491,13 @@ inline void
 {
     std::string filePath("/var/lib/bmcweb/ibm-management-console/configfiles/" +
                          fileID);
-    BMCWEB_LOG_DEBUG << "Removing the file : " << filePath << "\n";
+    BMCWEB_LOG_CRITICAL << "INFO:Removing the file : " << filePath << "\n";
```

Comment:
This log is not needed to be in the journal. Please revert

Reply:
@sunharis 
if I remove this log , path should be added in logs at 500 , 508 ,  516 separately 
500 BMCWEB_LOG_CRITICAL << "INFO:configFile removed!\n";
508 BMCWEB_LOG_ERROR << "File not removed!\n";
516 BMCWEB_LOG_ERROR << "File not found!\n";

Reply:
They are already error logs which will come in the journal. You can add the file path there to make them meaningful.